cmd/exchange_template/wrapper_file.tmpl

```diff
@@ -217,7 +217,7 @@ func ({{.Variable}} *{{.CapitalName}}) Run() {
 }
 
 // FetchTradablePairs returns a list of the exchanges tradable pairs
-func ({{.Variable}} *{{.CapitalName}}) FetchTradablePairs(ctx context.Context, asset asset.Item) ([]string, error) {
+func ({{.Variable}} *{{.CapitalName}}) FetchTradablePairs(ctx context.Context, a asset.Item) ([]currency.Pair, error) {
```

Comment:
A nice improvement! First brief look is that we can change the return type from []currency.Pair to currency.Pairs so that people can take advantage of all the helper methods